### PR TITLE
feat: cart

### PR DESCRIPTION
**Pull request by tlgimenes**

### CODE REVIEW

File: src/components/cart/CartSidebar/CartSidebar.module.css [hunk 1]
```diff
@@ -7,5 +7,53 @@
 }
 
 .container {
-  @apply p-7;
+  @apply p-0 h-full flex flex-col;
 }
+
+.x {
```
Comment (emersonlaurentino) [inline src/components/cart/CartSidebar/CartSidebar.module.css]: rename to close, is better 👀

File: src/components/cart/CartSidebar/CartSidebar.tsx [hunk 2]
```diff
@@ -1,29 +1,65 @@
 import { useCart, useGlobalUIState } from '@vtex/store-sdk'
 import React from 'react'
 import Drawer from 'src/components/ui/Drawer'
+import { X } from 'react-feather'
```
Comment (emersonlaurentino) [inline src/components/cart/CartSidebar/CartSidebar.tsx]:
```suggestion
import { X as CloseIcon } from 'react-feather'
```

File: src/components/cart/CartSidebar/CartSidebar.module.css (hunk 1 above)
Comment (mairatma) [reply src/components/cart/CartSidebar/CartSidebar.module.css]: Agreed 😅

File: src/components/common/Search/Search.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```
Comment (mairatma) [inline src/components/common/Search/Search.tsx]: Duplicated Search componente here again, you can probably just delete it, it's in `src/components/ui` in master.

**Review by mairatma — APPROVED** [review]

**Review by emersonlaurentino — APPROVED** [review]

**Pull request merged**